packages/types/src/documents/account/account.ts

```diff
@@ -79,14 +79,24 @@ export const isSelfHostAccount = (account: Account) =>
 export const isSSOAccount = (account: Account): account is SSOAccount =>
   account.authType === AuthType.SSO
 
-export interface SSOAccount extends Account {
-  pictureUrl?: string
-  provider?: string
-  providerType?: string
+export enum AccountSSOProviderType {
+  GOOGLE = "google",
+}
+
+export enum AccountSSOProvider {
+  GOOGLE = "google",
```

Comment:
What is the difference between this and the type? Should these field not be related?

Reply:
The sso fields on the account were modelled around the same fields on the budibase users, where: 
- providerType = the auth mechanism, in budibase this is either `oidc` or `google` 
- provider = the provider for the mechanism, always `google` for google type but can be any external identity provider when using oidc auth. 

The account fields are currently limited to google only as it's the only one we support.